DB_Models_and_Eloquent.md

```diff
@@ -312,7 +328,6 @@ class User extends Model
         if ($this->email) {
```

Comment:
```suggestion
        if (isset($this->attributes['email']) && ! is_null($this->attributes['email'])) {
```
The attributes array should be referenced to avoid hitting an accessor within a mutator.

Reply:
@KeithBush I have a little problem with this part.
Here we used `email` as an example, but in general, a column may get `null` value for the first time and it should not be changed.